<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Afb Websocket Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="pnlConnection" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblConnection">
         <property name="text">
          <string>Binder URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtConnection">
         <property name="text">
          <string>ws://localhost:1234/api?token=HELLO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnConnection">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tbFeatures">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tbpCalls">
       <attribute name="title">
        <string>Calls</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QComboBox" name="cbApi">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbVerb">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnCall">
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Call</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QsciScintilla" name="txtCallArg">
              <property name="toolTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QsciScintilla" name="txtCallResult">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tbpDebug">
       <attribute name="title">
        <string>Debug</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QListWidget" name="lstDebug"/>
          <widget class="QsciScintilla" name="txtDebug">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="pnlCustomText" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>45</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="txtCustomText"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnCustomText">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>txtConnection</sender>
   <signal>returnPressed()</signal>
   <receiver>btnConnection</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtCustomText</sender>
   <signal>returnPressed()</signal>
   <receiver>btnCustomText</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
